lib/web_ui/lib/src/engine/embedder.dart

```diff
@@ -21,26 +21,29 @@ import 'text_editing/text_editing.dart';
 import 'util.dart';
 import 'window.dart';
 
-class DomRenderer {
-  DomRenderer() {
-    if (assertionsEnabled) {
-      _debugFrameStatistics = DebugDomRendererFrameStatistics();
```

Comment:
We don't want these anymore?

Reply:
This has been superseded by `Instrumentation`.